README.md

```diff
@@ -16,6 +16,8 @@
 1. Navigate to Operate and check result.
 
 #### Migration Limitations
+- Async before/after wait states:
+  - C8 does not support async before/after wait states. Therefore, if you were to migrate an instance currently waiting asynchronously at an element in a C7 model, this instance would just continue without waiting in the equivalent C8 model. Please adjust your model's logic accordingly prior to migration
```

Comment:
🔧 Use full phrase. Maybe we can even add link to the docs: https://docs.camunda.org/manual/latest/user-guide/process-engine/transactions-in-processes/#asynchronous-continuations
```suggestion
  - C8 does not support Asynchronous continuation (before/after) wait states. Therefore, if you were to migrate an instance currently waiting asynchronously at an element in a C7 model, this instance would just continue without waiting in the equivalent C8 model. Please adjust your model's logic accordingly prior to migration
```